.github/workflows/coverity_scan.yml

```diff
@@ -0,0 +1,60 @@
+name: coverity-scan
+on:
+  workflow_dispatch: # because sometimes you just want to force a branch to have tests run
+  push:
+    branches:
+      - "**"
+    paths:
+      - "backend/**"
+      - .github/workflows/coverity_scan.yml
+  schedule:
+    - cron: '0 18 * * *' # Daily at 18:00 UTC
+
+defaults:
+  run:
+    working-directory: ./backend
+
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Install dependencies
+        run: pushd ../frontend/ && yarn install && popd
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Java Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: gradle-build-${{ hashFiles('*.gradle', 'gradle/dependency-locks/*') }}
+      - name: Download Coverity Build Tool
+        run: |
+          wget -q https://scan.coverity.com/download/java/linux64 --post-data "token=$TOKEN&project=CDCgov/prime-data-input-client" -O cov-analysis-linux64.tar.gz
+          mkdir cov-analysis-linux64
+          tar xzf cov-analysis-linux64.tar.gz --strip 1 -C cov-analysis-linux64
+        env:
+          TOKEN: ${{ secrets.COVERITY_TOKEN }}
+      - name: Build with cov-build
+        run: |
+          export PATH=`pwd`/cov-analysis-linux64/bin:$PATH
+          cov-build --dir cov-int --fs-capture-search ../frontend ./gradlew clean assemble
+      - name: Submit the result to Coverity Scan
+        run: |
+          tar czvf sr.tgz cov-int
+          curl \
+            --form token=$TOKEN \
+            --form email=nicholas.a.robison@omb.eop.gov \
```

Comment:
😕

Reply:
No context. Not sure what this means.

Reply:
Sorry—the fully-spelled-out version is that it doesn't seem great to have this tied to one person's e-mail.

Reply:
Agreed, but I'm not sure how to configure service accounts, or if they're even supported. Perhaps we can get an org account from CDC?

Reply:
Not sure what the email is￼ for here, but it seems like a distribution list email either at CDC or at USDS would be good in the longer term (really, at CDC).